lightly/models/modules/heads.py

```diff
@@ -0,0 +1,117 @@
+# TODO
+
+from typing import Union, List, Tuple
+
+import torch
+import torch.nn as nn
+from torch.nn.modules.batchnorm import BatchNorm1d
+
+
+class ProjectionHead(nn.Module):
+    """TODO
+    
+    """
+
+    def __init__(self, layers: List[Tuple[int, int, nn.Module, nn.Module]]):
```

Comment:
Is each list element a layer or other a block of (linear layer, batch norm layer, non-linear layer). If yes, than I would call it blocks.

Reply:
It's what you would call a block so I'll rename it.